alembic/env.py

```diff
@@ -1,11 +1,15 @@
 from logging.config import fileConfig
 
+import os
+import sys
 from sqlalchemy import engine_from_config
 from sqlalchemy import pool
 
 from alembic import context
 
-import emissionsapi.db
+home_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..')
+sys.path.insert(1, home_dir)
```

Comment:
Ok, you insert at this location and I need to read again how the python import order is :man_facepalming:

Reply:
This is basically just a “Hey, Python! Look into the current working directory first when searching for packages.”